packages/client-core/src/admin/components/Setting/index.tsx

```diff
@@ -147,7 +160,9 @@ const Sidebar = ({ selected, onChange }: SidebarProps) => {
   return (
     <List>
       {settingItems
-        .filter((item) => (item.scope ? userHasAccess(item.scope) : true))
+        .filter((item) =>
+          Array.isArray(item.scope) ? item.scope.find((scope) => userHasAccess(scope)) : userHasAccess(item.scope)
```

Comment:
This only seems to be checking if the user has access to any of the scopes in item.scope[], since Array.find returns the first item that returns true. Is this meant to be checking that all of the scopes in item.scope[] are ones the user has?

Reply:
Its meant to be checking one of the scopes.

i.e. `scope: ['settings:read', `${clientSettingPath}:read`]`

Here the client theme & settings will be accessible to user if he has either of the above scopes. Because in MT, the admin will only have `${clientSettingPath}:read` scope.